hearth/lib/hearth/context.rb

```diff
@@ -27,18 +32,15 @@ def initialize(options = {})
     # @return [Logger] An instance of the logger configured for the Client.
     attr_reader :logger
 
-    # @return [Hash] The hash of the original request parameters.
-    attr_reader :params
```

Comment:
Why are we removing params?

Reply:
It's not used, can add it back when we need it. We also will support unserialized params via input anyway. And also, we will use options instead of stuffing options in params. It's probably bad to keep around.